src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<ul><li>"precise" --&gt; "accurate"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Corrected in a6b9f26f